net/message/relay_message.go

```diff
@@ -0,0 +1,65 @@
+package message
+
+import (
+	"fmt"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+
+	"github.com/athanorlabs/atomic-swap/common/vjson"
+	contracts "github.com/athanorlabs/atomic-swap/ethereum"
+)
+
+// RelayClaimRequest implements common.Message for our p2p relay claim requests
+type RelayClaimRequest struct {
+	SFContractAddress ethcommon.Address          `json:"sfContractAddress" validate:"required"`
```

Comment:
```suggestion
	SwapContractAddress ethcommon.Address          `json:"swapContractAddress" validate:"required"`
```

potentially? in case of contract rename

Reply:
I changed it to `SwapFactoryAddress` to make it consistent with our naming elsewhere. That will help with the contract rename, as it will get changed in any global string replacement of `swapFactory` and `SwapFactory` into whatever the new name becomes.